core/app/controllers/uffizzi_core/api/cli/v1/projects/deployments/events_controller.rb

```diff
@@ -20,6 +20,11 @@ class UffizziCore::Api::Cli::V1::Projects::Deployments::EventsController <
   # @response 401 Not authorized
   # @response 404 Not found
   def index
+    unless resource_deployment.active?
```

Comment:
Please add comment to issue with subscribe how QA can reproduce this case and get this error.

Reply:
Added https://github.com/UffizziCloud/uffizzi_app/issues/236#issuecomment-1231788886